gradle/fips.gradle

```diff
@@ -5,9 +5,11 @@ import org.elasticsearch.gradle.testclusters.TestDistribution
 // Common config when running with a FIPS-140 runtime JVM
 if (BuildParams.inFipsJvm) {
 
+  String javaSecurityFilename = BuildParams.runtimeJavaDetails.startsWith('Oracle') ? 'fips_oracle_java.security' : 'fips_java.security'
```

Comment:
Is the behavior / code of jar verifier the same in all oracle jdks > 11 ?

Reply:
Yep it's the same.